panel/layout.py

```diff
@@ -313,3 +320,6 @@ def _get_model(self, doc, root=None, parent=None, comm=None):
         model = self._bokeh_model(**self._init_properties())
         self._link_params(model, ['width', 'height'], doc, root, comm)
         return model
+
+
+PaneBase._default_layout = Row
```

Comment:
Should this not be a public class Parameter?  Seems like something one could legitimately want to customize.